Added customization of manifest using svelte.config.js

SergeantWeb opened the pull request:
Simple feature added :

You can now customize manifest using svelte.config.json

## Usage example
Customize the manifest to name our extension `Test extension`
```javascript
// your-project/sveltekit.config.js
import adapter from 'sveltekit-adapter-browser-extension'

/** @type {import('@sveltejs/kit').Config} */
const config = {
	kit: {
		adapter: adapter({
			manifest: {
				name: "Test extension" // Customize app name in manifest
			},
		}),
		appDir: 'ext',
		prerender: {
			default: true
		}
	}
};

export default config;
```

SergeantWeb committed:
Added customization of manifest using svelte.config.js

antony commented:
I like the idea of this. I might need to create a new PR as the project now supports multiple manifest versions since this PR was made.

snyk-bot committed:
fix: upgrade cheerio from 1.0.0-rc.10 to 1.1.2

Snyk has created this PR to upgrade cheerio from 1.0.0-rc.10 to 1.1.2.

See this package in npm:
cheerio

See this project in Snyk:
https://app.snyk.io/org/sergeantweb/project/26a5656d-221d-4333-a074-9c70d6335a60?utm_source=github&utm_medium=referral&page=upgrade-pr

snyk-bot committed:
fix: upgrade @hapi/hoek from 9.2.1 to 9.3.0

Snyk has created this PR to upgrade @hapi/hoek from 9.2.1 to 9.3.0.

See this package in npm:
@hapi/hoek

See this project in Snyk:
https://app.snyk.io/org/sergeantweb/project/26a5656d-221d-4333-a074-9c70d6335a60?utm_source=github&utm_medium=referral&page=upgrade-pr

SergeantWeb committed:
Merge pull request #1 from SergeantWeb/snyk-upgrade-215140fab105dcb21b22158f2650c4f9

[Snyk] Upgrade cheerio from 1.0.0-rc.10 to 1.1.2

SergeantWeb committed:
Merge branch 'master' into snyk-upgrade-28581ba65b5cf35b525548179965ed2a

SergeantWeb committed:
Merge pull request #2 from SergeantWeb/snyk-upgrade-28581ba65b5cf35b525548179965ed2a

[Snyk] Upgrade @hapi/hoek from 9.2.1 to 9.3.0
